packages/notification-services-controller/CHANGELOG.md

```diff
@@ -48,7 +66,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- **BREAKING:** Bump depenency `firebase` from `^10.11.0` to `^11.2.0` ([#5196](https://github.com/MetaMask/core/pull/5196))
+- **BREAKING:** Bump dependency `firebase` from `^10.11.0` to `^11.2.0` ([#5196](https://github.com/MetaMask/core/pull/5196))
```

Comment:
Nit: Why is this breaking ?

Reply:
```suggestion
- Bump `firebase` from `^10.11.0` to `^11.2.0` ([#5196](https://github.com/MetaMask/core/pull/5196))

Reply:
hmm not sure why it was marked breaking, I was just fixing spelling.

Assuming this was because it was a major version bump from v10 to v11.

Reply:
The firebase package is a repo that uses subpath exports for smaller submodules.
We use the `firebase-messaging` module.

This version bump did not introduce anything breaking in `firebase-messaging`. So it is safe to mark this as non-breaking.

Reply:
removing the `**BREAKING**` field.